src/Symfony/Bridge/PhpUnit/Tests/CoverageListenerTest.php

```diff
@@ -16,7 +16,9 @@ public function test()
             $this->markTestSkipped('This test cannot be run on HHVM.');
         }
 
-        if (\PHP_VERSION_ID >= 70000) {
+        exec('type phpdbg', $output, $returnCode);
+
+        if (\PHP_VERSION_ID >= 70000 && 0 === $returnCode) {
```

Comment:
ping @nicolas-grekas : I did not have phpdbg on my machine, so I added that (IIRC, you added this part)

Reply:
would it work on windows?

Reply:
We don't care as this test is not run on windows ;)